wip/integrationguide.md

```diff
@@ -0,0 +1,110 @@
+
+```julia
+@defcomp component1 begin
+
+    # First define the state this component will hold
+    savingsrate = Parameter()
+
+    # Second, define the (optional) init function for the component
+    function init(p, v, d)
+    end
+
+    # Third, define the run_timestep function for the component
+    function run_timestep(p, v, d, t)
+    end
+
+end
+```
+
+## Constructing a Model
+
+In an effort to standardize the function naming protocol within Mimi, and to streamline it with the standards of Julia, several function names have been changed.  The table below lists a **subset** of these changes, focused on the exported API functions most commonly used in model construction.  
+
+| Old Syntax                | New Syntax                |
+| ------------------------  |:-------------------------:|
+|`connectparameter`         |`connect_parameter`        |
+|`setleftoverparameters`    |`set_leftover_params!`     |
+|`setparameter`             |`set_parameter!`           |
+|`adddimension`             |`add_dimension`            |
+|`setindex`                 |`set_dimension!`           |  
+
+Changes to various optional keyword arguments:
+
+ - `connect_parameter`:  In the case that a component parameter is connected to a variable from a prior timestep, it is necessary to use the `offset` keyword argument to prevent a cycle.  The offset value is an `Int` specifying the offset in terms of timesteps as below.
+
+```julia
+connect_parameter(mymodel, :TargetComponent=>:parametername, :SourceComponent=>:variablename, offset = 1)
+```
+- `add_component`:  Previously the optional keyword arguments `start` and `stop` could be used to specify times for components that do not run for the full length of the model. These arguments are now `first` and `last` respectively.
+
+```julia
+addcomponent(mymodel, ComponentC; first=2010, last=2100)
+```
+                        
+Finally, in order to finish connecting components, it is necessary to run `add_connector_comps` as below.
+
+```julia
+add_connector_comps(mymodel)
+
+```
+
+## Running a Model
+
+## Accessing Results
+
+## Plotting
+
+This release of Mimi does not include the plotting functionality previously offered by Mimi.  While the previous files are still included, the functions are not exported as efforts are made to simplify and improve the plotting associated with Mimi.  
+
+The new version does, however, include a new UI tool that can be used to visualize model results.  This `explore` funcion is described in the User Guide under **Advanced Topics**.
+
+## Advanced Topics
+
+### Timesteps and available functions
+
+As previously mentioned, some relevant function names have changed.  Below is a subset of such changes related to timesteps and available functions.
+
+| Old Syntax                | New Syntax                |
```

Comment:
Maybe add explanation for the change, i.e., to eliminate ambiguity about whether value given as "stop" was the final period run or the first period _not_ run. New naming is clearer.